src/main/flight/pid.c

```diff
@@ -1237,13 +1237,14 @@ bool pidAntiGravityEnabled(void)
 #ifdef USE_DYN_LPF
 void dynLpfDTermUpdate(float throttle)
 {
-    unsigned int cutoffFreq;
     if (pidRuntime.dynLpfFilter != DYN_LPF_NONE) {
+        float cutoffFreq;
```

Comment:
Changing the type to `float` makes usage in this function more natural and requires less type conversion.
However, if truncation of the value was desired we need to update this.
Same applies to changes in `gyro.c`.

Reply:
Truncation definitely isn't desired here. You found a small bug.